pkg/runtime/wfengine/backend.go

```diff
@@ -210,12 +257,18 @@ func (*actorBackend) DeleteTaskHub(context.Context) error {
 
 // GetActivityWorkItem implements backend.Backend
 func (be *actorBackend) GetActivityWorkItem(ctx context.Context) (*backend.ActivityWorkItem, error) {
-	// Wait for the workflow actor to signal us with some work to do
+	// Wait for the activity actor to signal us with some work to do
+	wfLogger.Debug("Actor backend is waiting for an activity actor to schedule an invocation.")
```

Comment:
These debug logs help make it easier to understand what's going on if a workflow or activity doesn't run when we expect it to (a problem that we've often seen but lacked clarity on what's actually stuck).